stimpacks/version/version.go

```diff
@@ -31,7 +32,7 @@ func (v *Version) Command(viper *viper.Viper) *cobra.Command {
 		Short: "Print the client version",
 		Long:  `Print the client version`,
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Printf("stim/%v\n", v.stim.GetVersion())
+			fmt.Printf("stim: %v\n", v.stim.GetVersion())
```

Comment:
I'd actually be ok removing `stim: ` entirely here - maybe will make it easier if needing to parse with bash etc..

Reply:
ok